pkg/util/trace.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"log"
+
+	"contrib.go.opencensus.io/exporter/ocagent"
+	"go.opencensus.io/trace"
+)
+
+// InitialiseTracing initialises trace, exporter and the sampler
```

Comment:
We prefer the "Initialize" spelling

Reply:
just for consistency's sake, I think we should spell this method as "InitializeTracing"